server/tests/integration_test.rs

```diff
@@ -0,0 +1,998 @@
+        body: cmd.into_bytes(),
+    };
+
+    rt.block_on(async {
+        let mut _result = client.do_action(Request::new(action)).await;
+
+        Ok(())
+    })
+}
+
+/// Generate a [`RecordBatch`] with the current timestamp, a random value and an optional tag.
+fn generate_random_message(tag: Option<String>) -> (RecordBatch) {
+    let timestamp = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("Can't generate the time.")
+        .as_micros() as i64;
+
+    let value = rand::thread_rng().gen_range(0..100) as f32;
+
+    if tag.clone().unwrap_or("".to_string()) == "" {
+        let message_schema = Schema::new(vec![
+            Field::new("timestamp", DataType::Timestamp(Millisecond, None), false),
+            Field::new("value", DataType::Float32, false),
+        ]);
+
+        (RecordBatch::try_new(
+            Arc::new(message_schema.clone()),
+            vec![
+                Arc::new(TimestampMillisecondArray::from(vec![timestamp])),
+                Arc::new(Float32Array::from(vec![value])),
+            ],
+        )
+        .expect("Could not generate RecordBatch."))
+    } else {
+        let message_schema = Schema::new(vec![
+            Field::new("timestamp", DataType::Timestamp(Millisecond, None), false),
+            Field::new("value", DataType::Float32, false),
+            Field::new("tag", DataType::Utf8, false),
+        ]);
+
+        (RecordBatch::try_new(
+            Arc::new(message_schema.clone()),
+            vec![
+                Arc::new(TimestampMillisecondArray::from(vec![timestamp])),
+                Arc::new(Float32Array::from(vec![value])),
+                Arc::new(StringArray::from(vec![tag.unwrap()])),
+            ],
+        )
+        .expect("Could not generate RecordBatch."))
+    }
+}
+
+/// Send messages to the ModelarDB server with Arrow Flight through the `do_put()` endpoint.
+fn send_messages_to_arrow_flight_server(
+    rt: &Runtime,
+    mut client: &mut FlightServiceClient<Channel>,
+    flight_data_vec: Vec<FlightData>,
+) -> std::result::Result<(), ()> {
+    rt.block_on(async {
+        let flight_data_stream = stream::iter(flight_data_vec);
+
+        let mut _streaming = client.do_put(flight_data_stream).await;
+
+        Ok(())
+    })
+}
+
+/// Flush the data in the StorageEngine to disk through the `do_action()` endpoint.
+fn flush_data_to_disk(rt: &Runtime, fsc: &mut FlightServiceClient<Channel>) -> Result<()> {
+    let action = Action {
+        r#type: "Flush".to_owned(),
+        body: vec![],
+    };
+    let request = Request::new(action);
+
+    rt.block_on(async {
+        fsc.do_action(request).await?.into_inner().message().await?;
+        Ok(())
+    })
+}
+
+/// Execute a query on the ModelarDB server through the `do_get()`endpoint.
+fn execute_query(
+    rt: &Runtime,
+    fsc: &mut FlightServiceClient<Channel>,
+    query: &str,
+) -> Result<Vec<RecordBatch>> {
+    rt.block_on(async {
+        //Execute query
```

Comment:
The comments in this function are missing initial spaces and punctuation.

Reply:
Fixed.

Reply:
The comments are still all missing periods.

Reply:
Fixed.